data/language/en-GB.txt

```diff
@@ -3618,6 +3618,8 @@ STR_6511    :Zero G Roll (right)
 STR_6512    :Large Zero G Roll (left)
 STR_6513    :Large Zero G Roll (right)
 STR_6514    :Invalid height!
+STR_6515    :Classic Wooden Roller Coaster
+STR_6516    :Classic Wooden Roller Coaster desc.
```

Comment:
I came up with an idea for the Classic Wooden Roller Coaster's description:
**_A fast and rough roller coaster built on wooden track with many ‘air time’ hills and twisting turns with a shallower banked curve angle than later roller coaster models_**

Reply:
Hm, I’m not sure about the “twisting turns” bit, as older woodies are not terribly good at them. It seems to me they are usually more out and back, and with more focus on hills than twisting turns.